resources/builder.inc.sh

```diff
@@ -1279,7 +1279,15 @@ _builder_do_build_deps() {
       $builder_debug \
       $_builder_build_deps \
       --builder-deps-built "$_builder_deps_built" \
-      --builder-dep-parent "$THIS_SCRIPT_IDENTIFIER"
+      --builder-dep-parent "$THIS_SCRIPT_IDENTIFIER" && (
```

Comment:
... out of curiosity, how did we end up with a subshell for the original form of the command at this point?

Reply:
This arises because of `if builder_start_action ...; then`, I think!